buildenv/jenkins/variables/defaults.yml

```diff
@@ -542,48 +552,12 @@ x86-64_linux_criu:
   build_env:
     cmd:
       11: 'source /home/jenkins/set_gcc7.5.0_env'
-      17: 'source /home/jenkins/set_gcc7.5.0_env'
-      18: 'source /home/jenkins/set_gcc7.5.0_env'
-      next: 'source /home/jenkins/set_gcc7.5.0_env'
+      17: 'source /home/jenkins/set_gcc_10.3.0_env'
```

Comment:
This should be done the same way as the other platforms, set 10.3 as the default.